app/models/manageiq/providers/google/network_manager/refresh_parser.rb

```diff
@@ -180,7 +180,7 @@ def get_cloud_networks
 
       def subnetworks
         unless @subnetworks
-          @subnetworks = @connection.list_aggregated_subnetworks.to_h[:items].flat_map { |_, v| v[:subnetworks] }
+          @subnetworks = @connection.list_aggregated_subnetworks.to_h[:items].values.flat_map { |v| v[:subnetworks] }
```

Comment:
@agrare, I couldn't do here what you've [suggested](https://github.com/ManageIQ/manageiq-providers-google/pull/54/files#r192153467). The `:subnetworks` is not an method but a key in Hash so we can't be calling it like:
```ruby
@connection.list_aggregated_subnetworks.to_h[:items].values.flat_map(&:subnetworks)
```

I've tried also this:
```ruby
@connection.list_aggregated_subnetworks.to_h[:items].values.pluck(:subnetworks).flatten
```
Despite it's being a bit better readable, it performs about twice slower than the `flat_map` version (due to the `flatten`). So I've left the `flat_map` here.

Reply:
:+1: it was the `.values` instead of `{ |_, v|` that I most cared about